test/templates/form-message/index.njk

```diff
@@ -4,27 +4,27 @@
 
 {% block content %}
 
-<h3>Default</h3>
+<h3>Success</h3>
 {% set example %}
-    <span class="fd-popover__body fd-popover__body--no-arrow fd-form-message" aria-hidden="true" id="popoverD1">Normal message</span>
```

Comment:
why are you removing these classes?

Reply:
because you don't need them. This is not self-contained